types/kraken-js/index.d.ts

```diff
@@ -5,22 +5,23 @@
 // TypeScript Version: 2.2
 
 import { Express } from 'express';
+import { EventEmitter } from 'events';
 
 declare function k(options?: k.Options | string): Express;
 
 declare namespace k {
-    interface Kraken extends Express {
-        once(name: string, cb: (err: Error | null, result?: any) => any): void;
+    interface Kraken extends Express, EventEmitter {
         kraken: Kraken;
     }
 
     interface Options {
-        protocols?: object;
+        protocols?: any;
```

Comment:
`any` also allows literals, where `object` does not, are you sure this is what you want?

Reply:
you're right, object is what I actually need, thanks!